src/test/java/de/qabel/desktop/hockeyapp/CrashesClientTest.java

```diff
@@ -35,37 +33,47 @@
         "  at CLASS.METHOD(FILE:LINE)\n" +
         "  at CLASS.METHOD(FILE:LINE)";
 
-    private String expectedFormattedStacktrace = "Package: de.qabel.desktop\n" +
-        "Version: 1.1\n" +
-        "OS: Linux / amd64 / 4.4.0-21-generic\n" +
-        "Manufacturer: Oracle Corporation\n" +
-        "Model: 1.8.0_91\n" +
-        "Date: " + new Date() + "\n" +
-        "Stacktrace: XCEPTION REASON STRING\n" +
-        "  at CLASS.METHOD(FILE:LINE)\n" +
-        "  at CLASS.METHOD(FILE:LINE)\n" +
-        "  at CLASS.METHOD(FILE:LINE)\n" +
-        "  at CLASS.METHOD(FILE:LINE)\n" +
-        "  at CLASS.METHOD(FILE:LINE)\n" +
-        "  at CLASS.METHOD(FILE:LINE)\n" +
-        "  at CLASS.METHOD(FILE:LINE)\n" +
-        "  at CLASS.METHOD(FILE:LINE)\n" +
-        "  at CLASS.METHOD(FILE:LINE)\n" +
-        "ANOTHER EXCEPTION REASON STRING\n" +
-        "  at CLASS.METHOD(FILE:LINE)\n" +
-        "  at CLASS.METHOD(FILE:LINE)\n" +
-        "  at CLASS.METHOD(FILE:LINE)\n" +
-        "  at CLASS.METHOD(FILE:LINE)";
+    private Date now = new Date();
+
+
     private String feedback = "this is just a String for the stacktrace";
 
+
     @Before
     public void setUp() throws Exception {
-        versionClient.setVersion(new HockeyAppVersion(1, requestBuilder.getAppVersion()));
+        versionClient.setVersion(new HockeyAppVersion(requestBuilder.getAppVersion()));
     }
 
     @Test
     public void createLog() throws IOException {
-        String formattedStacktrace = client.createLog(stacktrace);
+
+        String operatingSystemInformation = System.getProperty("os.name") + " / " + System.getProperty("os.arch") + " / " + System.getProperty("os.version");
+        String manufacturer = System.getProperty("java.vendor");
+        String model = System.getProperty("java.version");
```

Comment:
Use hardcoded strings for these 3 properties. Tests should look the same on all devices.